tests/test_date.py

```diff
@@ -2,90 +2,158 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from types import MethodType
+
+from collections import OrderedDict
+
```

Comment:
No, that is not what i said. What i meant is order and blank lines are not set according to PEP-8